pkg/pillar/cmd/zedmanager/updatestatus.go

```diff
@@ -33,6 +33,7 @@ func updateAIStatusUUID(ctx *zedmanagerContext, uuidStr string) {
 		log.Functionf("updateAIStatusUUID status change %d for %s",
 			status.State, uuidStr)
 		publishAppInstanceStatus(ctx, status)
+		publishAppInstanceSummary(ctx)
```

Comment:
Just a question to be sure I understand what's happening. Am I right that we are adding the call of the `publishAppInstanceSummary` function whenever there is a chance that the amount of running apps is changed?

Reply:
Yes, but the underlying pubsub package checks if there is any change to the struct before sending the updated info to the subscribers. So we don't need explicit checks in zedmanager to check if any of the summary counters changed.